packages/clay-management-toolbar/src/ClayManagementToolbar.soy

```diff
@@ -248,13 +252,15 @@
 	{@param? handleCloseMobileSearchClick_: any}
 	{@param? handleCreationButtonClicked_: any}
 	{@param? handleFilterDoneButtonClick_: any}
+	{@param? handleInfoButtonClicked_: any}
 	{@param? handleOpenMobileSearchClick_: any}
 	{@param? handleSearchSearchClick_: any}
 	{@param? handleSelectPageCheckboxChanged_: any}
 	{@param? handleSortingButtonClicked_: any}
 	{@param? handleViewTypeClicked_: any}
 	{@param? hideFiltersDoneButton: bool}
 	{@param? id: string}
+	{@param? infoButton: bool}
```

Comment:
For consistency with the rest of the params I think this should be named `showInfoButton`